pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -276,6 +276,7 @@ func (l *podNIC) PlugPhase1() error {
 			return createCriticalNetworkError(err)
 		}
 
+		domainIface := bindMechanism.generateDomainIfaceSpec()
```

Comment:
This continues the prev discussion on whether the return value should be a pointer or not. As this is the only usage, I fail to understand your reasoning to return the value and then pass its address to `setCachedInterface`. It is never used as a pure value and based on the implementation, it can never be a nil.

I will not insist on this, but it does raises question marks when reading (as if there is a hidden reason to this).